h5bp/ssl/policy_modern.conf

```diff
@@ -26,8 +26,8 @@
 #     $ssl_early_data variable should be used:
 #       proxy_set_header Early-Data $ssl_early_data;
 #
-#     The application should return response code 425 for anything that
-#     could contain user supplied data.
+#     The application should return response code 425 ("Too Early") for anything
```

Comment:
```suggestion
#     The application should return response code 425 "Too Early" for anything
```

Reply:
Fixed in https://github.com/h5bp/server-configs-nginx/pull/232/commits/863c0fb9bbcd16636096c3d27c594e20a769668a